src/manager/BaseMapFishPrintManager.js

```diff
@@ -103,6 +103,34 @@ export class BaseMapFishPrintManager extends Observable {
    */
   customParams = {};
 
+  /**
+   * Custom parameters which can be additionally set on map to determine its
+   * special handling while printing.
+   *
+   * The list of allowed properties is as follows:
+   *  * areaOfInterest
+   *  * bbox
+   *  * center
+   *  * rotation
+   *  * scale
+   *  * useNearestScale
+   *  * dpiSensitiveStyle
+   *  * useAdjustBounds
+   *  * layers
+   *  * width
+   *  * longitudeFirst
+   *  * projection
+   *  * dpi
+   *  * zoomToFeatures
+   *  * height
+   *
+   * Please refer to http://mapfish.github.io/mapfish-print-doc/attributes.html#!map
+   * for further details.
+   *
+   * @type {Object}
+   */
+  customMapParams = {};
```

Comment:
As this is not supported by the version 2 yet and the doc is matching the version 3, please move the attribute to the `MapFishPrintV3Manager`.

Reply:
You're right, moved it in 9464b43